config/packages/ezplatform_site_factory.yaml

```diff
@@ -1,7 +1,11 @@
 ez_platform_site_factory:
-    enabled: false
-#    templates:
-#        site_template:
-#            siteaccess_group: site_factory_group
-#            name: Example Site
-#            thumbnail: https://placekitten.com/500/500
```

Comment:
Actual needed size is not 500x500...

Reply:
Because http://localhost:8000/admin/site/list is the one deforming the most the image, 160x95 seems to be the better choice

![160x95](https://placekitten.com/160/95)

Reply:
https://github.com/ezsystems/ezplatform-ee/pull/138